src/rpc/fees.cpp

```diff
@@ -155,8 +153,6 @@ static RPCHelpMan estimaterawfee()
         },
         [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
         {
-            RPCTypeCheck(request.params, {UniValue::VNUM, UniValue::VNUM}, true);
```

Comment:
Small topic about this:
The last arg from `RPCTypeCheck` was `fAllowNull`, which was true here (so the two args were nullable). Now.. if we go to the arg definition, `conf_target` has `RPCArg::Optional::NO`. Which means the opposite.